cdk/script/ci

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
```

Comment:
Not directly related to this PR, but I wonder if our starter kit should assume GHA here and drop this kind of script with e.g. nvm setup.